tests/DerConverter.Tests/Asn/DerAsnPrintableStringTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using DerConverter.Asn;
+using NUnit.Framework;
+using System.Linq;
+
+namespace DerConverter.Tests.Asn
+{
+    [TestFixture]
+    public class DerAsnPrintableStringTests
+    {
+        [Test]
+        public void DerAsnPrintable_Parse_ShouldDecodeCorrectly()
+        {
+            var data = new byte[]
+            {
+                0x13, 0x02, 0x55, 0x53
+            };
+
+            var type = DerAsnPrintableString.Parse(data);
+            Assert.That(type is DerAsnPrintableString, Is.True);
+
+            Assert.That(type.Value, Is.EqualTo(data.Skip(2).ToArray()));
+        }
+
+        [Test]
+        public void DerAsnPrintable_GetBytes_ShouldEncodeCorrectly()
```

Comment:
`DerAsnPrintableString_GetBytes_ShouldEncodeCorrectly`

Reply:
Oh, good catch. Fixed it.